products/README.md

```diff
@@ -0,0 +1,10 @@
+# helpwave Products
+
+## helpwave task
+
+[Further reading](./tasks/README.md)
+
+
```

Comment:
```suggestion
```

Reply:
@MaxSchaefer shouldn't we leave that as base tructure?

Reply:
It's just that one line he suggests to remove.